addons/html_editor/static/src/main/movenode_plugin.js

```diff
@@ -405,16 +406,40 @@ export class MoveNodePlugin extends Plugin {
             const [position, focusElelement] = this._currentDropHintElementPosition;
             this._currentDropHintElementPosition = undefined;
             const previousParent = movableElement.parentElement;
+
+            const isFocusInsideList = ["UL", "OL"].includes(focusElelement?.parentElement?.tagName);
+            const isMovableLI = movableElement.tagName === "LI";
+            if (isMovableLI && !isFocusInsideList) {
```

Comment:
To me `isMovableLI` sounds more confusing let's just directly use `movableElement.tagName === "LI";`
```suggestion
            if (movableElement.tagName === "LI" && !isFocusInsideList) {
```